### ISSUE TITLE
guard-tork does not exit gracefully

### ISSUE THREAD

**Issue by aaronjensen** [sunaku/guard-tork#2]
When I ctrl+c guard, this happens:

```
/Users/./.rvm/gems/ruby-1.9.3-p194/gems/tork-19.0.0/lib/tork/server.rb:28:in `select': Interrupt
    from /Users/./.rvm/gems/ruby-1.9.3-p194/gems/tork-19.0.0/lib/tork/server.rb:28:in `block in loop'
    from /Users/./.rvm/gems/ruby-1.9.3-p194/gems/tork-19.0.0/lib/tork/server.rb:26:in `catch'
    from /Users/./.rvm/gems/ruby-1.9.3-p194/gems/tork-19.0.0/lib/tork/server.rb:26:in `loop'
    from /Users/./.rvm/gems/ruby-1.9.3-p194/gems/tork-19.0.0/lib/tork/master.rb:27:in `loop'
    from /Users/./.rvm/gems/ruby-1.9.3-p194/gems/tork-19.0.0/bin/tork-master:126:in `<top (required)>'
    from /Users/./.rvm/gems/ruby-1.9.3-p194/bin/tork-master:19:in `load'
    from /Users/./.rvm/gems/ruby-1.9.3-p194/bin/tork-master:19:in `<main>'
    from /Users/./.rvm/gems/ruby-1.9.3-p194/bin/ruby_noexec_wrapper:14:in `eval'
    from /Users/./.rvm/gems/ruby-1.9.3-p194/bin/ruby_noexec_wrapper:14:in `<main>'
```

Occasionally it doesn't happen, and sometimes the sock files are left in my home dir and tork will fail to start w/ addrinuse

**Comment by sunaku** [comment]
I'm okay with seeing that error, but I agree that socket files being left
behind is a bug.

Is it just master that's leaving socket files behind?

On my system, sending SIGINT and SIGTERM to a ruby 1.9.3 process correctly
calls the ensure clause, where the socket files are cleaned up.

**Comment by aaronjensen** [comment]
I just saw master, yes

**Comment by sunaku** [comment]
Socket file clean-up has been solved in Tork version 19.8.1 (2014-06-28). :ticket:
